### DISCUSSION TITLE
How to train with custom energy and force label?

### DISCUSSION THREAD

**Discussion by naveenmohandas** [MDIL-SNU/SevenNet#114] [Q&A]
Hi, 
 I was attempting to fine tune the Sevennet model, I have the input file in the .xyz format which can be read by the ASE. After spending sometime in the documentation I couldn't figure out how to pass custom energy labels when using the 
`sevenn input.yaml -s` 

Could you let me know how I can pass the energy and force labels to fine-tune the  model ?

**Comment by YutackPark** [comment]
This feature is introduced recently: #89 , and sorry for lack of documentation. The approach we did internally was writing custom scripts to assign EFS values with the `SingPointCalculator` of ASE (#61 )

From the version 0.10.0, custom energy, force, and stress keys can be assigned using the `energy_key`, `force_key`, and `stress_key` arguments.

for the `sevenn_inference` and `sevenn_graph_build`, append `--kwargs energy_key='my_energy' force_key='my_force'`.
For the training (actually same as graph_build), inside the `input.yaml`,
```yaml
data:
  data_format_args:
    energy_key: my_energy
    force_key: my_force
```

This _**Should**_ work, but personally, it feature is relatively new and may face unexpected problems... Please raise an issue if you face any problems with above approaches. Note that units and conventions (especially stress sign and Voigt notation) should same as that can be obtained from other ASE calculators.
